platform/macos/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 * Adjusted when and how the camera transition update and finish callbacks are called, fixing recursion bugs. ([#11614](https://github.com/mapbox/mapbox-gl-native/pull/11614))
 * Fixed an issue preventing nested key path expressions get parsed accordingly to the spec. ([#11959](https://github.com/mapbox/mapbox-gl-native/pull/11959))
 * Fixed race conditions that could cause crashes when re-using `MGLMapSnapshotter` or using multiple snapshotters at the same time. ([#11831](https://github.com/mapbox/mapbox-gl-native/pull/11831))
+* Fixed crash in `-[MGLStyle localizeLabelsIntoLocale:]` on iOS 9.3 (attempting to access a property that was introduced in iOS 10.0 and used in Darwin code) ([#12123](https://github.com/mapbox/mapbox-gl-native/pull/12123))
```

Comment:
s/iOS 9.3/macOS 10.11/
s/iOS 10.0/macOS 10.12/
The way we use “Darwin” in this repository is a bit misleading – we don’t actually do any kernel programming – so we avoid mentioning that term in public-facing documentation.

Reply:
Good to know - how do we normally reference code that is shared between iOS and macOS? "shared platform code"?

Reply:
Updated in https://github.com/mapbox/mapbox-gl-native/pull/12146

Reply:
> how do we normally reference code that is shared between iOS and macOS

We don’t tend to acknowledge the existence of other platforms on the iOS changelog.